.circleci/config.yml

```diff
@@ -907,66 +899,6 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
-  test-e2e-firefox-snaps:
-    executor: node-browsers
-    parallelism: 4
```

Comment:
Should we add more concurrent jobs for the main E2Es then? Since we are introducing a bunch of new tests?